docs/variants/pc_engines/hardware-matrix.md

```diff
@@ -20,3 +50,18 @@ coreboot port on the PC Engines apu2 family of mainboards.
     | **TPM**                        | PC Engines TPM.4A module                         |
     | **Power Supply**               | 12V power supply                                 |
     | **Power Control**              | RTE v1.0.0                                       |
+=== "apu6"
+
+    | Component                      | Description                                      |
+    |--------------------------------|--------------------------------------------------|
+    | **CPU**                        | AMD GX-412TC SOC                                 |
+    |                                | Passive cooling to enclosure                     |
+    | **RAM**                        | 2 GB DDR3 non-ECC (soldered)                     |
+    | **SSD**                        | Apacer ST170 60GB                                |
+    | **Flash memory**               | Winbond W25Q64JV                                 |
+    | **USB pendrives**              | SanDisk Ultra USB 3.0 32 GB                      |
```

Comment:
Can we use sticks @BeataZdunczyk has retrieved from the client for APU4 and APU6?

Reply:
Done.

Reply:
I don't see a change in the USB device @pkubaj . It could be helpful for a reviewer if you could link a commit with the given change. 
We should change the hardware matrix for apu4 and apu6, and add 'SWISSBIT USB Flash Drive U-56n' or something similar. It is exactly this one: https://pl.farnell.com/swissbit/sfu3008gc2ae1to-i-ge-1ap-std/usb-flash-drive-usb-3-1-8gb-pslc/dp/3527084

Reply:
Indeed, I didn't push that change, done now.